### PR TITLE
feat: update site footer

### PR DESCRIPTION
**Pull request by mamieorine**
Fixes https://github.com/GoogleChrome/web.dev/issues/9372

Changes proposed in this pull request:

- Update site footer following the design provided by Dan

### CODE REVIEW

**Review by devnook — APPROVED** [review]
Please wait for TW approval before merge.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**
